src/security/Handshake.cc

```diff
@@ -535,9 +535,13 @@ Security::HandshakeParser::parseHello(const SBuf &data)
     return false; // unreached
 }
 
-void
-Security::HandshakeParser::ParseCertificate(const SBuf &raw, Security::CertPointer &pCert)
+/// Creates and returns a new certificate by parsing a DER-encoded X509
+/// structure.
```

Comment:
I would remove ` new` and merge the two comment lines. Minor violations of the 72- or 80-character rule can be better than creating [widows](https://www.fonts.com/content/learning/fontology/level-2/text-typography/rags-widows-orphans) like this one.